src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -70,6 +70,9 @@ export var FeatureManager = VirtualGrid.extend({
       if (supportedFormats && supportedFormats.indexOf('geoJSON') !== -1) {
         this.service.options.isModern = true;
       }
+      if (error) {
```

Comment:
i can't repro the errors (even with semistandard `8.0.0`), but i think the change below would be sufficient and preferable to logging directly to the console.
```js
if (!error && supportedFormats && supportedFormats.indexOf('geoJSON') !== -1) {
```